src/stores/spaces.ts

```diff
@@ -10,7 +12,8 @@ export const useSpacesStore = defineStore('spaces', {
     loadingMore: false,
     loaded: false,
     hasMoreSpaces: true,
-    spaces: [] as Space[]
+    spaces: [] as Space[],
+    spacesStarred: useStorage(`${pkg.name}.spaces-starred`, [] as string[])
```

Comment:
```suggestion
    starredSpacesIds: useStorage(`${pkg.name}.spaces-starred`, [] as string[])
```
`starredSpacesIds` might be more easy to understand what is inside.